tests/transformation_tests/pathfinders/resources/extracted_validated_data.py

```diff
@@ -0,0 +1,180 @@
+import numpy as np
+import pandas as pd
+
+financial_completion_date = pd.DataFrame({"Pathfinder Financial Completion Date": [pd.Timestamp("2001-01-01")]})
```

Comment:
With the current flow all data will be `str` type as we don't do any type "coercion"

Reply:
Perhaps we should still add types to the `pa.Column` definitions and coerce the data.. we could run the fund specific validation on the non-coerced data so that we know we know we won't be missing any un-coercible data. Then, if all checks pass then we can return the coerced data to transformation. I presume it would be useful to have the data correctly typed for some transformations (particularly for `datetime`s?)